apps/explorer/src/pages/transaction-result/programmable-transaction-view/Inputs.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiCallArg } from '@mysten/sui.js';
+
+import { AddressLink, ObjectLink } from '~/ui/InternalLink';
+import { TableHeader } from '~/ui/TableHeader';
+
+interface Props {
+    inputs: SuiCallArg[];
+}
+
+export function Inputs({ inputs }: Props) {
+    if (!inputs?.length) {
+        return null;
+    }
+
+    return (
+        <>
+            <TableHeader>Inputs</TableHeader>
+            <ul className="flex flex-col gap-y-3">
+                {inputs.map((input, index) => {
+                    if (typeof input !== 'object') {
```

Comment:
Can this still happen?

Reply:
Yep, @mamos-mysten found it here https://github.com/MystenLabs/sui/pull/9395#issuecomment-1478452531

Reply:
We can merge this without this resolved but should work on figuring this out ASAP